noir-projects/aztec-nr/authwit/src/account.nr

```diff
@@ -54,6 +56,17 @@ impl AccountActions {
         )
     }
 
+    pub fn initialize_account_contract(self) {
```

Comment:
I'd rename this to something like "execute_fee_payload_for_init", since it's not actually initializing the contract.